honeycomb-kernels/src/grisubal/model.rs

```diff
@@ -6,38 +6,39 @@
 
 // ------ IMPORTS
 
-use std::collections::HashSet;
-
+use crate::grisubal::GrisubalError;
 use honeycomb_core::attributes::AttrSparseVec;
 use honeycomb_core::prelude::{
     AttributeBind, AttributeUpdate, CoordsFloat, DartIdentifier, OrbitPolicy, Vertex2,
 };
-
 use vtkio::{
     model::{CellType, DataSet, VertexNumbers},
     IOBuffer, Vtk,
 };
 
-use crate::grisubal::grid::GridCellId;
-use crate::grisubal::GrisubalError;
-#[cfg(doc)]
-use honeycomb_core::prelude::CMap2;
-
 // ------ CONTENT
 
-/// Post-processing clip operation.
+/// Structure used to index the overlapping grid's cases.
 ///
-/// Note that the part of the map that is clipped depends on the orientation of the original geometry provided as
-/// input.
-#[derive(Default)]
-pub enum Clip {
-    /// Clip elements located on the left side of the oriented boundary.
-    Left,
-    /// Clip elements located on the right side of the oriented boundary.
-    Right,
-    /// Keep all elements. Default value.
-    #[default]
-    None,
+/// Cells `(X, Y)` take value in range `(0, 0)` to `(N, M)`,
+/// from left to right (X), from bottom to top (Y).
+#[derive(PartialEq, Clone, Copy)]
+pub struct GridCellId(pub usize, pub usize);
+
+impl GridCellId {
+    /// Compute the [Manhattan distance](https://en.wikipedia.org/wiki/Taxicab_geometry) between
+    /// two cells.
+    pub fn man_dist(lhs: &Self, rhs: &Self) -> usize {
```

Comment:
This name is confusing. `L1` or `distance1` or something like this is more readable if you want to keep it short.

Reply:
renamed to `l1_dist`